rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/indices.put_index_template/15_composition.yml

```diff
@@ -286,3 +286,73 @@
 
   - is_false: purple-index.mappings.properties.nested.include_in_root
   - is_true: purple-index.mappings.properties.nested.include_in_parent
+
+---
+"Index template ignore_missing_component_template valid":
+  - skip:
+      version: " - 8.6.99"
+      reason: "index template v2 ignore_missing_component_template config not available before 8.7"
+      features: allowed_warnings
+
+  - do:
+      cluster.put_component_template:
+        name: red
+        body:
+          template:
+            mappings:
+              properties:
+                foo:
+                  type: keyword
+
+  - do:
+      indices.put_index_template:
+        name: blue
+        body:
+          index_patterns: ["purple-index"]
+          composed_of: ["red", "blue"]
+          ignore_missing_component_templates: ["blue"]
+          #data_stream: { }
+
+  - do:
+      # I would like to create a data stream and not and index. But did not see the tests for it to copy
```

Comment:
@dakrone Is there a way to create a data stream? I know, in theory outcome should be the same but in the end most users will use this with data streams so I rather test it directly with this scenario.

Reply:
Yes, you can create one with this API: https://www.elastic.co/guide/en/elasticsearch/reference/master/indices-create-data-stream.html (use `indices.create_data_stream` with `name: purple-index` in your yml test)

Reply:
Great, thanks.